test/lev-top-nav/Index.test.js

```diff
@@ -26,4 +26,9 @@ describe('component', () => {
     let link = root.findByProps({className: "page-link", href: '/history'});
     expect(link.props.children).toEqual('History');
   });
+
+  it('contains a link to the user activity report', () => {
+    let link = root.findByProps({className: "page-link", href: 'https://lev.homeoffice.gov.uk/audit/user-activity/'});
```

Comment:
@aarongill1 - I might be tempted to add the `target: '_blank'` prop in here too, as that is relatively important, but it's up to you...

Reply:
Done